var/spack/repos/builtin/packages/bazel410/package.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import re
+
+from spack.package import *
+
+
+class Bazel410(Package):
+    """Bazel is an open-source build and test tool similar to Make, Maven, and
+    Gradle. It uses a human-readable, high-level build language. Bazel supports
+    projects in multiple languages and builds outputs for multiple platforms.
+    Bazel supports large codebases across multiple repositories, and large
+    numbers of users."""
+
+    # TODO: This package only exists because different packages in the Bazel ecosystem
+    # require different versions of Bazel. Once the concretizer is capable
+    # of concretizing build dependencies separately, this package should be removed.
```

Comment:
Can you remind me which packages this is? TF now has new releases in the package that support newer bazel, and tensorboard no longer builds with bazel, so I'm wondering if this is still needed.

Reply:
If memory serves, it was jaxlib.

Reply:
Jaxlib requires exactly Bazel 4.1.0, but TF can also build with this version of Bazel now too, right? If you replace `depends_on('bazel410')` with `depends_on('bazel@4.1.0')` does anything fail to concretize?

Reply:
Confirmed that everything concretizes if I remove `bazel410` and depend on `bazel@4.1.0`. So this package can be removed.